pkg/runtime/runtime.go

```diff
@@ -852,6 +850,9 @@ func (a *DaprRuntime) subscribeTopic(parentCtx context.Context, name string, top
 			path:       routePath,
 			pubsub:     name,
 		}
+		policyRunner := resiliency.NewRunner[any](ctx,
+			a.resiliency.ComponentInboundPolicy(name, resiliency.Pubsub),
```

Comment:
Isn't this the same as policyDef (from L730)?

Reply:
Yeah, correct, I will use the already defined policyDef here.